tests/dump_tests/module_tests/fdb_test.py

```diff
@@ -0,0 +1,196 @@
+import json
+import os
+import sys
+import jsonpatch
+import unittest
+import pytest
+from deepdiff import DeepDiff
+from mock import patch
+from dump.helper import create_template_dict
+from dump.plugins.fdb import Fdb
+from dump.match_infra import MatchEngine, ConnectionPool
+from swsscommon.swsscommon import SonicV2Connector
+
+# Location for dedicated db's used for UT
+module_tests_path = os.path.dirname(__file__)
+dump_tests_path = os.path.join(module_tests_path, "../")
+tests_path = os.path.join(dump_tests_path, "../")
+dump_test_input = os.path.join(tests_path, "dump_input")
+fdb_files_path = os.path.join(dump_test_input, "fdb")
+
+# Define the mock files to read from
+dedicated_dbs = {}
+dedicated_dbs['APPL_DB'] = os.path.join(fdb_files_path, "appl_db.json")
+dedicated_dbs['ASIC_DB'] = os.path.join(fdb_files_path, "asic_db.json")
+dedicated_dbs['STATE_DB'] = os.path.join(fdb_files_path, "state_db.json")
+
+
+def populate_mock(db, db_names):
```

Comment:
This can be removed and can be imported from dump.helper. Use this ref: https://github.com/Azure/sonic-utilities/commit/4e132c1c86c515076815a64cf22aaf8e0101de84 to modify the test based on the refactored design